app/assets/stylesheets/custom.scss

```
@@ -184,3 +184,39 @@ input {
     border-bottom: 1px solid $gray-lighter;
   }
 }
+
+.stats {
+  overflow: auto;
+  margin-top: 0;
+  padding: 0;
+  a {
+    float: left;
+    padding: 0 10px;
+    border-left: 1px solid $gray-lighter;
+    color: gray;
+    &:first-child {
```

Thêm 1 line trống trước các selector gần nhau nhé bạn 
```suggestion
    color: gray;
    
    &:first-child {
```